package.json

```diff
@@ -10,9 +10,14 @@
     "index-terms": "node ./scripts/index/term/index.js",
     "index-issues": "node ./scripts/index/issues/index.js",
     "security-check": "npm audit",
-    "test": "./node_modules/.bin/nyc mocha --reporter xunit tests",
+    "test": "mocha --reporter xunit tests",
+    "test-coverage": "./node_modules/.bin/nyc npm test",
     "lint": "./node_modules/.bin/eslint .",
-    "lint-fix": "./node_modules/.bin/eslint --fix ."
+    "lint-fix": "./node_modules/.bin/eslint --fix .",
+    "postinstall": "opencollective-postinstall"
+  },
+  "collective": {
+    "url": "https://opencollective.com/webpack"
```

Comment:
Do these Open Collective things still need to happen since we're disabling Open Collective in the manifest?

Reply:
It is not going to hurt, and all it does display a message when you do npm install "...please donate..."